gcc internal compiler error: Segmentation fault

```
build Polymake
  Building CxxWrap ─→ `~/.julia/packages/CxxWrap/lDNAy/deps/build.log`
  Building CMake ───→ `~/.julia/packages/CMake/nSK2r/deps/build.log`
  Building Polymake → `~/Berlin-dev/updating_cxxwrap/dev/Polymake/deps/build.log`
 Resolving package versions...
┌ Error: Error building `Polymake`: 
│ -- Found Julia executable: /opt/julia-1.3.0/bin/julia
│ -- Julia_VERSION_STRING: 1.3.0
│ -- Julia_INCLUDE_DIRS:   /opt/julia-1.3.0/include/julia
│ -- Julia_LIBRARY_DIR:    /opt/julia-1.3.0/lib
│ -- Julia_LIBRARY:        /opt/julia-1.3.0/lib/libjulia.so.1
│ -- JULIA_HOME:           /opt/julia-1.3.0/bin
│ -- Julia_LLVM_VERSION:   v6.0.1
│ -- Julia_WORD_SIZE:      64
│ -- Configuring done
│ -- Generating done
│ -- Build files have been written to: /home/kalmar/Berlin-dev/updating_cxxwrap/dev/Polymake/deps/src
│ Scanning dependencies of target polymake
│ [  6%] Building CXX object CMakeFiles/polymake.dir/polymake.cpp.o
│ [ 12%] Building CXX object CMakeFiles/polymake.dir/polymake_arrays.cpp.o
│ [ 18%] Building CXX object CMakeFiles/polymake.dir/polymake_bigobjects.cpp.o
│ [ 25%] Building CXX object CMakeFiles/polymake.dir/polymake_caller.cpp.o
│ /home/kalmar/Berlin-dev/updating_cxxwrap/dev/Polymake/deps/src/polymake_caller.cpp: In instantiation of ‘funcall_type<VoidContext> polymake_call_function(const string&, const std::vector<std::__cxx11::basic_string<char> >&, jlcxx::ArrayRef<_jl_value_t*>) [with bool VoidContext = false; funcall_type<VoidContext> = pm::perl::PropertyValue; std::string = std::__cxx11::basic_string<char>]’:
│ /home/kalmar/Berlin-dev/updating_cxxwrap/dev/Polymake/deps/src/polymake_caller.cpp:67:77:   required from here
│ /home/kalmar/Berlin-dev/updating_cxxwrap/dev/Polymake/deps/src/polymake_caller.cpp:47:12: internal compiler error: Segmentation fault
│    47 |     return static_cast<funcall_type<VoidContext>>(function());
│       |            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
│ Please submit a full bug report,
│ with preprocessed source if appropriate.
│ See <https://bugs.archlinux.org/> for instructions.
│ make[2]: *** [CMakeFiles/polymake.dir/build.make:102: CMakeFiles/polymake.dir/polymake_caller.cpp.o] Error 1
│ make[2]: *** Waiting for unfinished jobs....
│ make[1]: *** [CMakeFiles/Makefile2:73: CMakeFiles/polymake.dir/all] Error 2
│ make: *** [Makefile:130: all] Error 2
│ ERROR: LoadError: failed process: Process(`make -j2`, ProcessExited(2)) [2]
│ 
│ Stacktrace:
│  [1] pipeline_error at ./process.jl:525 [inlined]
│  [2] #run#565(::Bool, ::typeof(run), ::Cmd) at ./process.jl:440
│  [3] run(::Cmd) at ./process.jl:438
│  [4] top-level scope at /home/kalmar/Berlin-dev/updating_cxxwrap/dev/Polymake/deps/build.jl:151
│  [5] include at ./boot.jl:328 [inlined]
│  [6] include_relative(::Module, ::String) at ./loading.jl:1105
│  [7] include(::Module, ::String) at ./Base.jl:31
│  [8] include(::String) at ./client.jl:424
│  [9] top-level scope at none:5
│ in expression starting at /home/kalmar/Berlin-dev/updating_cxxwrap/dev/Polymake/deps/build.jl:151
└ @ Pkg.Operations /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.3/Pkg/src/backwards_compatible_isolation.jl:649
```
now this happens for both the BinaryProvider and locally build Polymake. @benlorenz, could you remind me what was the fix for this?

--- a/deps/src/CMakeLists.txt
+++ b/deps/src/CMakeLists.txt
@@ -12,6 +12,14 @@ include_directories(${MY_LIST})
 SET( CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} -std=c++14 ${polymake_cflags}" )
 SET( CMAKE_SHARED_LINKER_FLAGS  "${CMAKE_EXE_LINKER_FLAGS} ${polymake_ldflags}" )
 
+# avoid gcc 9 internal compiler error,
+# see https://gcc.gnu.org/bugzilla/show_bug.cgi?id=90998
+if(CMAKE_COMPILER_IS_GNUCC AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER 9.0
+      AND CMAKE_CXX_COMPILER_VERSION VERSION_LESS 9.3)
+   SET( CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} -Wno-conversion" )
+endif()
+
+
 file(GLOB polymake_SRC "*.cpp")
 add_library(polymake SHARED ${polymake_SRC})
 target_link_libraries(polymake JlCxx::cxxwrap_julia "${polymake_libs}")

